contrib/cirrus/shellcheck.sh

```diff
@@ -1,16 +0,0 @@
-#!/bin/bash
-
-set -eo pipefail
-
-# shellcheck source=./contrib/cirrus/lib.sh
-source $(dirname $0)/lib.sh
-
-cd $CIRRUS_WORKING_DIR
```

Comment:
I'm frantically identifying all the appropriate diffs to run. As part of that, I noticed that this `cd` was removed from the final `prebuild.sh`. Was that intentional? Is there a working-directory guarantee? I couldn't find one in `.cirrus.yml`.

Reply:
I'm not sure if it's documented anywhere, but it has always behaved this way.  In any case, it probably wouldn't hurt to put this into the new scripts just to be safe and obvious to maintainers.